### PR TITLE
CTCTRADERS-2023 Link the Backend to manage FE to make TAD pdf link available for status relased for transit

### PR DESCRIPTION
**Pull request by snehaprathipat1**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -184,9 +184,9 @@ viewDepartures.table.officeOfDeparture = Office of departure
 viewDepartures.table.status = Status
 viewDepartures.table.action = Action
 viewDepartures.table.action.viewErrors        = View and fix errors
-viewDepartures.table.action.viewCancellation       = View cancellation
+viewDepartures.table.action.viewCancellation  = View cancellation
 viewDepartures.table.action.viewHistory       = View history
-viewDepartures.table.action.viewPDF           = TAD PDF ({0})
+viewDepartures.table.action.viewPDF           = TAD PDF
```
Comment (ericohwotu) [inline conf/messages.en:189]: could that key be something like viewTadPDF as there is also the TSAD which might have a separate link

File: test/viewModels/DepartureStatusSpec.scala [hunk 2]
```diff
@@ -60,11 +60,11 @@ class DepartureStatusSpec extends SpecBase with Generators with ScalaCheckProper
     "include tad link on ReleasedForTransit status" in {
       forAll(arbitrary[Departure]) {
         departure =>
-          val dep             = departure.copy(status = "ReleasedForTransit")
+          val dep             = departure.copy(status = "ReleaseForTransit")
```
Comment (ericohwotu) [inline test/viewModels/DepartureStatusSpec.scala:63]: status name, is this the one we want to go with?

File: conf/messages.en (hunk 1 above)
Comment (snehaprathipat1) [reply conf/messages.en:189]: Couldn't disagree. The content for that will be updated in a separate ticket unless you want something sensible here now I will update that change.

File: test/viewModels/DepartureStatusSpec.scala (hunk 2 above)
Comment (snehaprathipat1) [reply test/viewModels/DepartureStatusSpec.scala:63]: Yes 👍

File: test/connectors/DeparturesMovementConnectorSpec.scala [hunk 3]
```diff
@@ -272,6 +273,46 @@ class DeparturesMovementConnectorSpec extends SpecBase with WireMockServerHandle
...
+      }
+
+      "must return other error status codes without exceptions" in {
+
+        val genErrorResponse = Gen.oneOf(300, 500).sample.value
+        val departureId      = DepartureId(0)
+
+        server.stubFor(
+          get(urlEqualTo(s"/$startUrl/movements/departures/${departureId.index}/transit-accompanying-document"))
+            .withHeader("Channel", containing("web"))
```
Comment (ericohwotu) [inline test/connectors/DeparturesMovementConnectorSpec.scala:303]: missing user agent header

File: app/config/FrontendAppConfig.scala [hunk 4]
```diff
@@ -57,6 +57,7 @@ class FrontendAppConfig @Inject()(configuration: Configuration) {
   lazy val referenceDataUrl: String   = configuration.get[Service]("microservice.services.reference-data").fullServiceUrl
   lazy val routerUrl: String          = configuration.get[Service]("microservice.services.testOnly-router").fullServiceUrl
   lazy val enrolmentKey: String       = configuration.get[String]("keys.enrolmentKey")
+  lazy val manageService: String = configuration.get[String]("microservice.services.manage")
```
Comment (ericohwotu) [inline app/config/FrontendAppConfig.scala:60]: Key should be `appName` not `microservice.services.manage`

File: conf/application.conf [hunk 5]
```diff
@@ -78,6 +78,13 @@ microservice {
           startUrl = "transit-movements-trader-at-destination"
       }
 
+       manage {
```
Comment (ericohwotu) [inline conf/application.conf:81]: This change isn't needed

**Review by ericohwotu — APPROVED** [review]

**Pull request merged**